docs/cli.md

```diff
@@ -0,0 +1,100 @@
+---
+title: CLI Reference
+---
+This document describes the Prisma CLI commands, arguments, and options.
+
+## Overview
+
+The `wasp` command can be called from command line once installed. 
+When called without arguments, it will display its command usage and help document:
+
+```
+USAGE
+  wasp <command> [command-args]
+
+COMMANDS
+  GENERAL
+    new <project-name>    Creates new Wasp project.
+    version               Prints current version of CLI.
+  IN PROJECT
+    start                 Runs Wasp app in development mode, watching for file changes.
+    db <db-cmd> [args]    Executes a database command. Run 'wasp db' for more info.
+    clean                 Deletes all generated code and other cached artifacts. Wasp equivalent of 'have you tried closing and opening it again?'.
+    build                 Generates full web app code, ready for deployment. Use when deploying or ejecting.
+    telemetry             Prints telemetry status.
+
+EXAMPLES
+  wasp new MyApp
+  wasp start
+  wasp db migrate-dev
+
+Docs: https://wasp-lang.dev/docs
+Discord (chat): https://discord.gg/rzdnErX
+
+```
+
+## Commands
+### General
+ - `wasp new <project-name>` creates new Wasp project.
```

Comment:
Here I would skip the output of the command, and instead I would add an extra line saying that it is a simple hello world project which can be run immediately. You could also mention that directory with the provided proejct-name will be created. And provide an example below, smth like `wasp new MyFirstProject`. I find examples are always useufl, even when trivial.

Reply:
Agree!